packages/aws-cdk-lib/aws-rds/lib/instance.ts

```diff
@@ -865,6 +920,7 @@ abstract class DatabaseInstanceNew extends DatabaseInstanceBase implements IData
       domain: this.domainId,
       domainIamRoleName: this.domainRole?.roleName,
       networkType: props.networkType,
+      caCertificateIdentifier: props.caCertificate ? props.caCertificate.identifier : CaCertificate.RDS_CA_2019.identifier,
```

Comment:
Default question, see above.

```suggestion
      caCertificateIdentifier: props.caCertificate ? props.caCertificate.identifier : undefined,
```

Btw, we also could implement a `toString()` method on `CaCertificate`. This is works as well though.